DoCoMoQaReq request
---
bool success
DoCoMoQaRes response 
